client/src/includes/breadcrumbs.test.js

```diff
@@ -0,0 +1,70 @@
+import initCollapsibleBreadcrumbs from './breadcrumbs';
+
+describe('initCollapsibleBreadcrumbs', () => {
+  it('should do nothing if there is no breadcrumbs container', () => {
+    jest.spyOn(document, 'addEventListener').mockImplementation(() => {});
```

Comment:
I recommend you have another read of the docs

* https://jestjs.io/docs/jest-object#jestspyonobject-methodname
* If you do want to call `spyOn` within each test you should call `mockRestore` at the end of each test
* Also, when you mock the implementation, you will force the mock do do nothing and this may cause issues with your tests.

So.. get the value of the spy instead and remove the mockImplementation

```suggestion
   const spy = jest.spyOn(document, 'addEventListener'));
```

Then at the end of your test call ` spy.mockRestore();`

Reply:
Done